Makefile

```diff
@@ -36,6 +42,21 @@ test: venv ## Run tests
 integration-test: ## Run integration tests
 	./scripts/run_integration_tests.sh
 
+.PHONY: build-wheel
+build-wheel: venv ## build distributable wheel
+	pip install wheel
```

Comment:
Should this be installed in the venv? (./venv/bin/pip install wheel). The question stands for the python/twine runs too.